docs/contributing.rst

```diff
@@ -67,6 +67,8 @@ The following tools are there to help you:
 
 Integration
 ~~~~~~~~~~~
+Mac OS X users: Run `./tests/mac-integration.sh` to configure the
```

Comment:
`mac-integration.sh` doesn't match the file name below which is what I think you mean.